internal/cli/trigger_apply.go

```diff
@@ -94,17 +94,22 @@ func (c *TriggerApplyCommand) Run(args []string) int {
 		}
 
 		// Trigger target
-		if diffTrigger.Workspace != nil {
+		if diffTrigger.Workspace != nil && c.flagWorkspace == "" {
```

Comment:
```suggestion
		if c.flagWorkspace == "" {
		    if diffTrigger.Workspace != nil {
		       c.flagWorkspace = diffTrigger.Workspace.Workspace
		       } else {
		       c.flagWorkspace = "default"
		   }
```
To make that flow cleaner, and optionally refactor the triggers below to have the flag check as the first conditional for uniformity?

Reply:
We'll still have to do this workspace flag check later, because inside this if-statement we're only setting these values if we're updating a trigger URL and one already exists, i.e. the check on line 83. So in the creation case, we'll still need the check below if workspace was not set.